sql/hive/src/main/scala/org/apache/spark/sql/hive/test/TestHiveContext.scala

```diff
@@ -37,39 +38,36 @@ import org.apache.spark.sql.execution.CacheTableCommand
 import org.apache.spark.sql.hive._
 import org.apache.spark.sql.hive.execution.HiveNativeCommand
 import org.apache.spark.util.Utils
-import org.apache.spark.{SparkConf, SparkContext}
 
 /* Implicit conversions */
 import scala.collection.JavaConversions._
 
-// SPARK-3729: Test key required to check for initialization errors with config.
-object TestHive
```

Comment:
Yes, we can avoid it in our own tests, but I know for a fact that it is used externally.